src/imex.rs

```diff
@@ -706,6 +711,19 @@ async fn export_database(
         .await
 }
 
+/// Sets `Config::DeleteServerAfter` to "never" if needed so that new messages are present on the
+/// server after a backup restoration or available for all devices in multi-device case.
+/// NB: Calling this after a backup import isn't reliable as we can crash in between, but this is a
+/// problem only for old backups, new backups already have `DeleteServerAfter` set if necessary.
+async fn adjust_delete_server_after(context: &Context) -> Result<()> {
```

Comment:
wondering if the naming used on android isn't more on point, so eg. `assume_multi_device()` or `adjust_delete_for_multi_device()`.

but as always, naming is hard, i am totally fine with the current name as well, description is needed anyways :)

Reply:
The reason is that "adjusting" the setting has sense even for single-device in order to make backups really useful, otherwise new messages are deleted since the backup was done

Reply:
good you point that out. if we want to get down on space usage on chatmail further, eg. by tweaking this option on _import_ only, we should keep that possible drawback mind